recipes/drm4g/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "drm4g" %}
+{% set version = "3.0.11" %}
+{% set sha256 = "ece23c282d1d8fb82b34a8885166676984fd846da7ff644edc5fc174364de2a0" %}
+{% set build_number = 0 %}
```

Comment:
Please only the name and version placeholder, everything else should be hardcoded. Otherwise the bot might get tripped up.

Reply:
I have only added the parameterized build number, the sha256 was added by grayskull package.

Anyway, both will be hard-coded as requested.

Reply:
Ahh strange, Grayskull should only add the following:

```yaml
{% set name = "drm4g" %}
{% set version = "3.0.11" %
```

If you see this again, can you please file an issue with Grayskull?